terragrunt/env/staging/ecs/terragrunt.hcl

```
@@ -0,0 +1,77 @@
+terraform {
+  source = "../../../aws//ecs"
+}
+
+dependencies {
+  paths = ["../iam", "../network", "../ecr", "../load_balancer", "../databse"]
```

Sorry - this could have been me, but database is mispelled 
```suggestion
  paths = ["../iam", "../network", "../ecr", "../load_balancer", "../database"]
```